build/scripts/GitHub/GitHubIssue.class.psm1

```diff
@@ -0,0 +1,48 @@
+using module .\GitHubAPI.class.psm1
+
+class GitHubIssue {
+    $IssueId
+    $Repository
+    $Issue
+
+    #Constructor
+    hidden GitHubIssue([int] $IssueId, [string] $Repository) {
+        $this.IssueId = $IssueId
+        $this.Repository = $Repository
+
+        $gitHubIssue = gh api "/repos/$Repository/issues/$IssueId" -H ([GitHubAPI]::AcceptJsonHeader) -H ([GitHubAPI]::GitHubAPIHeader)| ConvertFrom-Json
+        if (-not $gitHubIssue) {
```

Comment:
This does not work as `$gitHubIssue` is still an object when the issue is not found.